.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: Test
+on:
+  push:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+  pull_request:
+    branches: [main]
+    paths: [src/**, action.yml, .github/wiki/**, .github/workflows/test.yml]
+# We don't want cancel-in-progress because we want each test (if there are
+# multiple at a given time) to still attempt to push to the wiki. This should
+# not cause any hiccups on the wiki since any and all tests are publishing in
+# the same content.
+concurrency: test-${{ github.ref }}
```

Comment:
Are we sure about this? 2 events occurring at around the same time can easily cause a conflict AFAIK.
1. job `foo` and `bar` run at close to same time
2. `foo` and `bar` fetch the wiki's remote, and HEAD is now at `abc123` for both of them
3. both make commits, now 2 different commits have the same parent `abc123`
4. both push, one is guaranteed to fail

Reply:
ohhh i thought they force-pushed to the wiki, in which case there'd be no difference.

Reply:
fixed in https://github.com/spenserblack/actions-wiki/pull/38/commits/368b3982f8567271a4333cc6f85bdf322a94aade